src/frontend/komponenter/Felleskomponenter/IkonKnapp/IkonKnapp.tsx

```diff
@@ -37,6 +38,7 @@ const IkonKnapp: React.FC<IProps> = ({
             type="flat"
             mini={mini}
             kompakt={true}
+            {...props}
```

Comment:
Jeg synes alltid ...props er litt skummelt, siden jeg føler man mister litt kontrollen på hva som sendes rundt omkring. Jeg har dog ikke lest meg noe opp på om dette faktisk er best practice eller om det finnes andre bedre måter. Du trenger ikke endre det, men følte bare for å lufte tanken!

Reply:
Ja, problemet er at før så fikk ikke konsumenten bruke de faktiske propsene til knappen - kun de vi har definert som er litt uheldig.

Reply:
Ja, enig i at det er uheldig. Eventuelt kan man liste opp, men da blir det jo en veldig lang liste som også er kjipt. Kanskje jeg skal lese litt på dette en dag 🤓